docs/wip-plugins.md

```diff
@@ -370,6 +388,121 @@ Returns `true` if the channel can be moderated by the current user.
 
 Returns `true` if the current user is a VIP in the channel.
 
+#### `HTTPMethod` enum
+
+This table describes HTTP methods available to Lua Plugins. The values behind
+the names may change, do not count on them. It has the following keys:
+
+- `Get`
+- `Post`
+- `Put`
+- `Delete`
+- `Patch`
+
+#### `HTTPResult`
```

Comment:
I prefer the name Response over Result

I know we internally call it Result, but most other libraries call them `Request` and `Response`